jenkins/config/kubernetes-cap.yaml

```diff
@@ -0,0 +1,16 @@
+groovy:
+  - script: >
+      import jenkins.model.Jenkins;
+
+      CONTAINER_CAP = 15;
+
+      cloud = Jenkins.instance.clouds.find { it.name == "openshift" };
+      if (cloud == null) {
+        throw new Exception("Failed to find cloud openshift");
+      }
+
+      if (cloud.containerCap != CONTAINER_CAP) {
```

Comment:
How often does this code get triggered? If we go in and set it manually to something lower (like `5`) will it get reset back and when?

Reply:
It runs on Jenkins start, so whenever Jenkins gets restarted.